metadata-models/src/main/pegasus/com/linkedin/glossary/GlossaryTermInfo.pdl

```diff
@@ -47,4 +47,9 @@ record GlossaryTermInfo {
    * A key-value map to capture any other non-standardized properties for the glossary term
    */
   customProperties: map[string, string] = { }
+
+  /**
+   * Schema definition of the glossary term
+   */
+  schemaDefinition: optional string
```

Comment:
what is the difference between schemaDefinition and definition?

Reply:
schemaDefinition is the raw schema for Glossary terms whereas definition is the description for that term.

Reply:
would `rawSchema` make more sense as a title then?

Reply:
Renamed it to `rawSchema`